yaki_database/create_tables.sql

```diff
@@ -0,0 +1,189 @@
+-- Begin with initiating the sequence for IDs regarding the users, captains, teams, team mates, locations and declarations
```

Comment:
change queries order. Script should be runnable in one shot. For the moment some tables are created too late